examples/wasm_node/test/test.js

```diff
@@ -99,7 +99,7 @@ describe('Key generation / derivation', function() {
 
     it('Key Derive Invalid Path', () => {
         assert.throws(
-            () => signer_wasm.key_derive(EXAMPLE_MNEMONIC, "m/44'/461'/a/0/1"),
+            () => signer_wasm.key_derive(EXAMPLE_MNEMONIC, "m/44'/461'/a/0/1", ""),
```

Comment:
Can we add a test with a "password" to be sure it actually works and changes the result?